### ISSUE TITLE
A11y_VS IDE (DougAm) Individual R&D Subscript_Authentication Mode_ScreenReader:When using the arrow keys to change the selection in the Authentication Mode combo box, screen reader is not announcing the change of selection.

### ISSUE THREAD

**Issue by Roopesh-T** [dotnet/winforms#7603]
### .NET version

.NET Framework Version 4.7.2
Operating System: Windows 11 Enterprise 21H2
Microsoft Visual Studio Enterprise 2022 (64-bit) - Int Preview
Version 17.4.0 Preview 2.0 [32815.475.main]

### Did it work in .NET Framework?

No

### Issue description

**GitHubTags:**#A11yMAS;#A11yTCS;#BM_VS IDE (DougAm) Individual R&D SubscriptionContainer_Win32_Jul2022;#VS IDE (DougAm) Individual R&D SubscriptionContainer;#Win32;#FTP;#DesktopApp;#A11ySev3;#AILimited;#WCAG1.3.1;#NVDA;#Jaws;#STP;#Element:Combobox;#Benchmark;#Linked:Bug1576845;#LinkedBug;

## Actual:
When using the arrow keys to change the selection in the Authentication Mode combo box, screen reader is not announcing the change of selection.

**Similar issue not observed in JAWS**
## Expected:
When using the arrow keys to change the selection in the Authentication Mode combo box, screen reader should be announcing the change of selection.
## User Impact:
Screen reader user will not be able to get the change of selection information and might accidentally change the selection of combo box.

### Steps to reproduce

1. Please do not close this bug. This bug should only be closed by TCS, C+AI Accessibility tester after bug verification
2. Open Visual Studio version 17.4.0 preview 1.0
3. Run NVDA
4. Create a new "Windows Forma App (.Net Framework" project 
5. Tab till "Properties" in "Solution Explorer" section and press enter
6. Tab till "Debug" tab item and press enter
7. Tab till "Use remote machine" checkbox and select the checkbox
8. Tab to the Authentication Mode combo box use arrow keys to change the section in the combo box and check if the screen reader is announcing the change in combo box.
![arrow keys bug blank winforms](https://user-images.githubusercontent.com/86601127/185085907-4f1c17d9-5cd0-41c4-ad50-21ffc7b6c42b.gif)
[VBWinformsNetFrameworkShowcaseBadComboBox.zip](https://github.com/dotnet/winforms/files/9358395/VBWinformsNetFrameworkShowcaseBadComboBox.zip)

**Comment by Tanya-Solyanik** [comment]
@Roopesh-T - this is not the right place to open this bug, It should be opened in AzDO

**Comment by davkean** [comment]
http://github.com/dotnet/project-system owns the above Debug UI.

**Comment by merriemcgaw** [comment]
Please note for future reference that the correct place for filing .NET Framework WinForms bugs is in the DevDiv AzDO instance with an Area Path of DevDiv\NET Fundamentals\Winforms\. Also note that I highly doubt that this is a WinForms .NET Framework issue if it is only observed in NVDA - I tested your application and it announces fine in Narrator as well. I also tested the VS UI in question with Accessibility Insights to confirm that not only is the correct UIA tree in place, but that the Value Pattern returns a different value each time the drop down is changed, and it does. This is 100% chance an NVDA bug.
